### ISSUE TITLE
Images selection or deletion from an inspection

### ISSUE THREAD

**Issue by scaspers** [DanielCaspers/DigitalInspection#108]
When a tech loads a picture that the service advisor feels would not be helpful they would like the option to delete or select which ones the customer will see.

**Comment by scaspers** [comment]
I loaded it on DI2 and it seems to error out sometimes.  First I uploaded 3 images.  Then I deleted one with no error. Then I went back in and tried to delete a second and got an error.  After clicking around the work order tabs a bit I went back in and deleted a second image successfully.  The I tried to delete the last and it errorred.  I logged out and back in went to delete the last image and it errorred.  I think this is the same error each time but this is the screen shot from the last attempt.
![image](https://user-images.githubusercontent.com/12421599/39456983-10c155f6-4cb0-11e8-9f64-49f34c61bbb3.png)
